src/memote/utils.jl

```diff
@@ -106,4 +107,19 @@ to_element(x::String) = begin
     elements[sym]
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+A helper function that returns the median upper and lower bounds in a tuple. If none can be calculated
+(-1000.0, 1000.0) are set as the default values.
+"""
+function median_bounds(model::MetabolicModel)
+    lower_bound, upper_bound = bounds(model)
+    lb_list = [element for element in lower_bound if element != 0.0]
+    ub_list = [element for element in upper_bound if element != 0.0]
+    isempty(lb_list) ? m_lower_bound = -1000.0 : m_lower_bound =  median(lb_list) 
+    isempty(ub_list) ? m_upper_bound = 1000.0 : m_upper_bound =  median(ub_list) 
+    return m_lower_bound, m_upper_bound
+end
+
```

Comment:
In which test is this function used? Did I miss it?

Reply:
it is used down below in testing the unbounded flux

Reply:
totally missed it 😳